docs/site/tutorials/introducing_x10.ipynb

```diff
@@ -301,6 +302,50 @@
       "execution_count": 0,
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cwNDwzS2QgS1",
+        "colab_type": "text"
+      },
+      "source": [
+        "Now, we wiil implement basic progress tracking and reporting.  All data is kept as tensors on the same device where training is run and calls `scalarized()` only during reporting to avoid unneccesary materialization of lazy tensors."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "eNzOdly3QY_P",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "struct Statistics {\n",
+        "    var correctGuessCount = Tensor<Int32>(0, on: Device.default)\n",
+        "    var totalGuessCount = Tensor<Int32>(0, on: Device.default)\n",
+        "    var totalLoss = Tensor<Float>(0, on: Device.default)\n",
+        "    var batches: Int = 0\n",
+        "    var accuracy: Float { Float(correctGuessCount.scalarized()) / Float(totalGuessCount.scalarized()) * 100 } \n",
```

Comment:
I'd recommend expanding this out onto the next two lines, to avoid the line length exceeding the width of the cell.

Reply:
Done.